phpBB/config/default/container/services_attachments.yml

```diff
@@ -0,0 +1,57 @@
+services:
```

Comment:
I don't really like duplicating a service definition... but it shouldn't harm that much here